lib/cppcheck.h

```diff
@@ -215,6 +215,8 @@ class CPPCHECKLIB CppCheck : ErrorLogger {
 
     unsigned int mExitCode;
 
+    bool mSuppressExitCode;
```

Comment:
I understand the logic better now. Please change the variable name to mSuppressInternalErrorFound